### ISSUE TITLE
Validator eats btor2 files

### ISSUE THREAD

**Issue by ChristianMoesl** [cksystemsteaching/selfie#182]
Is that intended behavior? 

<img width="625" alt="image" src="https://user-images.githubusercontent.com/17096190/86921063-d6844000-c12a-11ea-8f53-a75e0e2b7c92.png">

**Comment by Gotsiller** [comment]
This is not intended, but neither is the way you use the tool.
Validator takes a C* file as input.
In your case, validator assumes that the btor2 file you give it, is the temporary file it usually produces from the C* file.
Since temp files are deleted in the cleanup step, also your btor2 file is deleted.

**Comment by ckirsch** [comment]
@Gotsiller Does it make sense to run validator without the binary? Validation is then not possible, right?

**Comment by Gotsiller** [comment]
@ckirsch This is right. Validator needs to run the C* file on Selfie to verify that the error predicted by btormc actually occurs.

**Comment by ckirsch** [comment]
@ChristianMoesl Why do use the validator the way you do?

**Comment by ChristianMoesl** [comment]
I looked it up in the Makefile and clearly misunderstood it. If you use the validator in the right way, everything is ok.

Still I think it could be programmed a little bit more robust. All calls to `system()` are not checked for a correct error code.
That leads to problems, when for an example `btormc` is not installed on the machine. The validator will tell you in that case, that no error state was found, when in reality, `btormc` was not even executed.

But that is another topic :)
